x/emissions/keeper/keeper.go

```diff
@@ -1112,19 +1112,13 @@ func (k *Keeper) AppendInference(
 		return errors.New("inference already submitted")
 	}
 
+	// Get active inferers for topic
 	workerAddresses, err := k.GetActiveInferersForTopic(ctx, topic.Id)
 	if err != nil {
 		return errorsmod.Wrap(err, "error getting active inferers for topic")
 	}
```

Comment:
To avoid potential unnecessary queries, I'd move this below `GetLowestInfererScoreEma` since it's just used after getting the lowest score.

Reply:
Done